Training_1/Program.cs

```diff
@@ -3,12 +3,42 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program on T14 branch.
 // ------------------------------------------------------------------------------------------------
-namespace Training_1 {
-   internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+using static System.Console;
+namespace Training_1;
+
+internal class Program {
+   static void Main () {
+      Write ("Enter a number: ");
+      if (int.TryParse (ReadLine (), out int n)) {
+         var (minSum, convertedNumber) = MinTransform (n);
+         WriteLine ($"Minimum cost: {minSum}");
+         WriteLine ($"Transformed number: {convertedNumber}");
+      } else WriteLine ("Invalid input");
+   }
+
+   // Function to calculate the minimum transformation cost by changing all digits to a single digit by calculating the
+   // absolute difference between each digit and the target digit
+   static (int minSum, string convertedNumber) MinTransform (int n) {

```

Comment:
Tuple return variable names should start with uppercase
```suggestion
   static (int MinSum, string ConvertedNumber) MinTransform (int n) {
```